src/host-config.ts

```diff
@@ -0,0 +1,174 @@
+import * as Bluebird from 'bluebird';
+import * as _ from 'lodash';
+import * as mkdirCb from 'mkdirp';
+import { fs } from 'mz';
+import * as path from 'path';
+
+import Config = require('./config');
+import * as constants from './lib/constants';
+import { ENOENT } from './lib/errors';
+import { writeFileAtomic } from './lib/fs-utils';
+import * as systemd from './lib/systemd';
+
+const mkdirp = Bluebird.promisify(mkdirCb) as (
+	path: string,
+	opts?: any,
+) => Bluebird<mkdirCb.Made>;
+
+const redsocksHeader = `\
+base {
+	log_debug = off;
+	log_info = on;
+	log = stderr;
+	daemon = off;
+	redirector = iptables;
+}
+
+redsocks {
+	local_ip = 127.0.0.1;
+	local_port = 12345;
+\
```

Comment:
I don't think this `\` is necessary?

Reply:
Decaffeinate added these, I think you're right - not necessary.